users/management/commands/verify_user_education.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+import requests
+
+
+class Command(BaseCommand):
+
+    help = "Verify User Education"
+
+    def add_arguments(self, parser):
+        parser.add_argument('id', type=int, nargs='?', help='Define a user id')
+        parser.add_argument('-a', '--all', type=bool, nargs='?', default=False, help="Get all users education status")
+
+    def handle(self, *args, **options):
+        id = options['id'] or ''
+        url = 'http://localhost:8000/users/{0}'.format(id)
+        response = requests.get(url=url)
```

Comment:
This command does not make sense. It is suppose to two types of input 1. id of the student to verify education or `all` string value in case of `all` it should iterate over all students in the system and pull their education records from and API and update each student data to mark them either verified or not verified.

Reply:
Done.